src/corehaptics.cs

```diff
@@ -405,6 +414,10 @@ interface CHHapticPattern {
 		[Wrap ("this (patternDefinition.GetDictionary ()!, out outError)")]
 		NativeHandle Constructor (CHHapticPatternDefinition patternDefinition, [NullAllowed] out NSError outError);
 
+		[TV (16,0), NoWatch, MacCatalyst (16,0), Mac (13,0), iOS (16,0)]
+		[Export ("initWithContentsOfURL:error:")]
+		NativeHandle Constructor (NSUrl url, [NullAllowed] out NSError outError);
```

Comment:
nit: remove out prefix

```suggestion
		NativeHandle Constructor (NSUrl url, [NullAllowed] out NSError error);
```